runtime/karura/src/lib.rs

```diff
@@ -570,6 +570,9 @@ parameter_types! {
 	pub DataDepositPerByte: Balance = deposit(0, 1);
 	pub const MaximumReasonLength: u32 = 8192;
 	pub const MaxApprovals: u32 = 30;
+
+	pub const SevenDays: BlockNumber = 7 * DAYS;
```

Comment:
this can be in runtime common

Reply:
the two runtimes have different block time - so i think this means this can't be in the common? (mandala has 6 seconds but karura has 12 seconds per block)

Reply:
Oh right. That's fine then.